contracts/ProjectRegistry.sol

```diff
@@ -276,33 +284,38 @@ contract ProjectRegistry is
     /**
      * @notice Indexer update its service status to ready with a specific deploymentId
      */
-    function startService(bytes32 deploymentId) external onlyIndexer {
-        ServiceStatus currentStatus = deploymentStatusByIndexer[deploymentId][msg.sender];
+    function startService(
```

Comment:
Can you leave original method so we can have smooth upgrade?

Reply:
Yeah, though about that before, in that case we want to keep both of them or just for transition